src/ClimbObc.c

```diff
@@ -94,7 +102,10 @@ static const MODULE_DEF_T Modules[] = {
 		MOD_INIT( sen_init, sen_main, NULL),
 		MOD_INIT( memInit, memMain, &MemoryInit),
 		MOD_INIT( gpsInit, gpsMain, &GpsInit),
-		MOD_INIT( app_init, app_main, NULL)
+		MOD_INIT( app_init, app_main, NULL),
+		MOD_INIT( thrInit, thrMain, &ThrInit),
+		MOD_INIT( app_init, app_main, NULL),
```

Comment:
seems to be duplicated line (from merge !?) -> pls. delete

Reply:
https://github.com/RobertK66/obc_1769_core/pull/34/commits/0e793dce1459ebc0f6b293d8c4a6936331faa6b2

Fixed